src/js/merkle-drop/ClaimFlow.js

```diff
@@ -328,7 +328,11 @@ function ClaimFlow() {
       return (
         <div className="columns is-centered">
           <div className="column is-three-fifths">
-            <ClaimSuccess txHash={txHash} confirmations={confirmations} />
+            <ClaimSuccess
+              txHash={txHash}
+              confirmations={confirmations}
+              amount={currentClaimAmount}
```

Comment:
This shows what our backend calculated what should be claimable, but not what was actually claimed. Please parse the event from the blockchain to be sure to show the actually claimed amount.

Reply:
:heavy_check_mark: